pkg/gitreceive/build.go

```diff
@@ -97,7 +100,7 @@ func build(conf *Config, s3Client *s3.S3, kubeClient *client.Client, builderKey,
 	bType := getBuildTypeForDir(tmpDir)
 	usingDockerfile := bType == buildTypeDockerfile
 
-	var procType pkg.ProcessType
+	procType := pkg.ProcessType{}
```

Comment:
I think this worked. Previously build hook from workflow use to bail out as Proctype is None.  This actually fixed it.